public/app/plugins/datasource/grafana-pyroscope-datasource/dataquery.cue

```diff
@@ -36,7 +36,7 @@ composableKinds: DataQuery: {
 				// Allows to group the results.
 				groupBy: [...string]
 				// Sets the maximum number of nodes in the flamegraph.
-				maxNodes?:        int64
+				maxNodes?:           int64
```

Comment:
Seems like this is why our squad is a reviewer here, at the same time I assume this is not intentional :)

Reply:
It's not! Not sure how that got there, it was probably a hook?

Reply:
Trying to rever the change on 42ad201 resulted in an empty commit, it's caused by a pre-commit hook.